examples/6.educational_pluto_notebook/simulation-example-solution.jl

```diff
@@ -277,73 +299,73 @@ The spin echo experiment has the advantage that the echo signal amplitud it is m
 
 For this section we will use the phantom `obj_t2star` and a new sequence `seq_se`.
 
-Our sequence consists of:
- - (4.1) A 90deg hard RF pulse
- - (4.2) A `Delay` of $$\mathrm{TE}/2$$ with a positive gradient (area `Ax`)
- - (4.3) A 180deg hard RF pulse
- - (4.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in $$\mathrm{TE}$$
- - (4.5) Create concatenating these blocks into a sequence called `seq_se`
- - (4.6) Plot `seq_se` and its k-space. Is the k-space the same as `seq_gre` in (2.3)?
+- (4.1) Create a sequence called `seq_se` with the following blocks:
+  * (4.1.1) A 90deg hard RF pulse
+  * (4.1.2) A `Delay` of $$\mathrm{TE}/2$$ with a positive gradient (area `Ax`)
+  * (4.1.3) A 180deg hard RF pulse
+  * (4.1.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in $$\mathrm{TE}$$
+ 
+ - (4.2) Plot `seq_se` and its k-space. Is the k-space the same as `seq_gre` in (2.3)?
```

Comment:
Revert numberings and text changes. The text before this already mentions `seq_se`. I intended to do something like "For this sequence we will need:" It is not necesarry to tell people to create that sequence, is obvious.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/253/commits/3fde402ca7a7613045f15f2c77564a4f5491fcfb

Reply:
"For this sequence we will need:"

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/253/commits/1b1e5f3d71a7e9f5cd135f14ac9ad9409722a5e9